python/nav/web/templates/styleguide.html

```diff
@@ -411,8 +411,8 @@ <h3 class="underline">Forms</h3>
       </ul>
 
       <p>
-        There are many customizations done to the crispy forms. Basically, if you
-        need to create forms - ask on mail/jabber/irc.
+        Use existing forms as examples for how to build your own.  If you have questions, ask them
+        on the NAV mailing lists or GitHub discussion forums.
```

Comment:
```suggestion
        on the NAV mailing list or GitHub discussion forum.
```

Reply:
Maybe we could even add a link to the discussion forum?

Reply:
We have multiple mailing lists.  GitHub discussions has what I perceive as multiple forums per repo.

Reply:
Contact details are documented elsewhere, I wouldn't want to repeat that here, but point to the relevant documentation.